appcenter-crashes/Crashes.d.ts

```diff
@@ -0,0 +1,60 @@
+export const enum UserConfirmation {
+  DONT_SEND = 0,
+  SEND = 1,
+  ALWAYS_SEND = 2
+}
+
+export interface ListenerMap {
+  onBeforeSending?: (report: ErrorReport) => void;
+  onSendingSucceeded?: (report: ErrorReport) => void;
+  onSendingFailed?: (report: ErrorReport) => void;
+  getErrorAttachments?: (report: ErrorReport) => Promise<ErrorAttachmentLog[]>;
+}
+
+export class ErrorAttachmentLog {
+  public static attachmentWithText(text: string, fileName: string): ErrorAttachmentLog;
+  public static attachmentWithBinary(
+    data: object,
+    fileName: string,
+    contentType: string
+  ): ErrorAttachmentLog;
+}
+
+export interface Device {
+  sdkName: string;
+  sdkVersion: string;
+  model: string;
+  oemName: string;
+  osName: string;
+  osVersion: string;
+  osBuild: string;
+  osApiLevel: number;
+  locale: string;
+  timeZoneOffset: number;
+  screenSize: string;
+  appVersion: string;
+  carrierName: string;
+  carrierCountry: string;
+  appBuild: string;
+  appNamespace: string;
+}
+
+export interface ErrorReport {
+  id: string;
+  threadName: string;
+  appErrorTime: string;
+  appStartTime: string;
+  exception: string;
```

Comment:
can you double check type here?

Reply:
`errorReportMap.putString("exception", Log.getStackTraceString(errorReport.getThrowable()));`
`errorReportMap.putString("exceptionReason", errorReport.getThrowable().getMessage());`

Both in https://github.com/Microsoft/AppCenter-SDK-React-Native/blob/develop/appcenter-crashes/android/src/main/java/com/microsoft/appcenter/crashes/AppCenterReactNativeCrashesUtils.java#L32 which is called from https://github.com/Microsoft/AppCenter-SDK-React-Native/blob/develop/appcenter-crashes/android/src/main/java/com/microsoft/appcenter/crashes/AppCenterReactNativeCrashesModule.java#L77

iOS same story: https://github.com/Microsoft/AppCenter-SDK-React-Native/blob/develop/appcenter-crashes/ios/AppCenterReactNativeCrashes/AppCenterReactNativeCrashesUtils.m#L110 called from https://github.com/Microsoft/AppCenter-SDK-React-Native/blob/develop/appcenter-crashes/ios/AppCenterReactNativeCrashes/AppCenterReactNativeCrashes.m#L111